src/app/client/src/app/modules/dashboard/components/course-progress/course-progress.component.html

```diff
@@ -60,15 +60,23 @@
               <b>{{resourceService?.frmelmnts?.lbl?.participants}}&nbsp;:
               </b>  &nbsp;&nbsp; {{dashboarData.count}}
             </span>
-            <span *ngIf="dashboarData.completedCount" class="course-completed pl-15">
+            <span *ngIf="dashboarData.completedCount!==null" class="course-completed pl-15">
               <b>{{resourceService?.frmelmnts?.lbl?.completedCourse}}&nbsp;:
               </b>
               &nbsp;&nbsp;{{dashboarData.completedCount}}
             </span>
-            <!-- This needs to be developed -->
-            <!-- <span class="course-completed ml-auto">
-              <span class="font-italic">This report was last updated on</span>&nbsp;:&nbsp;dd/mm/yyyy
-            </span> -->
+            
+            <span class="course-completed ml-auto" *ngIf="dashboarData.reportUpdatedOn && dashboarData.reportUpdatedOn!==null">
+              <span class="font-italic">{{resourceService?.frmelmnts?.lbl?.reportUpdatedOn}}</span>&nbsp;:&nbsp;{{dashboarData.reportUpdatedOn | dateFormat :'DD/MM/YYYY'}}
+            </span>
+          </div>
+        </div>
+        <div class="sb-toaster sb-toast-info sb-mw-100" *ngIf="dashboarData &&  dashboarData.count>10000" >
```

Comment:
why is this message in toast?

Reply:
it is info because of that we are putting inside toast info. after discussion with UI team i have put like this ,